libs/console/shell/src/lib/shell/shell.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="sidebar">
-  <div class="application-title">Carrier Portal</div>
+  <div class="application-title">CoverME.gov</div>
```

Comment:
I think it should be better if we add this value on TenantConfig like "friendly name" or something similar, that way if it's needed to deploy on DC or on another customer it will be future proof

Reply:
This has been done, it now uses the translation file properly.